l10n/en/products/vpn/platforms/shared.ftl

```diff
@@ -0,0 +1,44 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/products/vpn/desktop/
+
+# Shared
+vpn-cta-headline = Let's get started
+vpn-shared-cta = See pricing & availability
+vpn-shared-day-guarantee = 30-day money-back guarantee
+vpn-shared-what-youll-get = What you’ll get with { -brand-name-mozilla-vpn }:
+
+vpn-shared-feature-item-protection = Protection for up to { -vpn-available-devices } devices
```

Comment:
NIt: use a variable for device numbers

Reply:
[See above comment.](https://github.com/mozilla/bedrock/pull/9974#discussion_r592049267)